types/types.go

```diff
@@ -426,6 +427,9 @@ func (l Labels) Add(key, value string) Labels {
 	return l
 }
 
+// SSHConfig is a mapping type for SSH build config
+type SSHConfig map[string]string
```

Comment:
I wonder `SSHConfig` should be a `[]SSHKey`, the latter declared as struct { ID, Path }` (to match the spec)